gui/build.py

```diff
@@ -118,19 +113,48 @@ def parse_args():
 
     bundle = subparsers.add_parser('bundle',
                                    help='bundle the application (freeze)')
+    if IS_WIN:
+        default_python_path = 'C:\\Python27\\Lib'
+        bundle.add_argument('--nsis-path', dest='nsis_path',
+                            default='C:\\NSIS',
+                            help='your NSIS path on the'
+                            ' system(default: %(default)r)')
+    else:
+        default_python_path = '/usr/lib/python2.7'
+
     bundle.add_argument('--python-path', dest='python_path',
-                        default='C:\\Python27',
+                        default=default_python_path,
                         help='your python path on the'
                         ' system(default: %(default)r)')
-    bundle.add_argument('--nsis-path', dest='nsis_path',
-                        default='C:\\NSIS',
-                        help='your NSIS path on the'
-                        ' system(default: %(default)r)')
     bundle.set_defaults(func=do_bundle)
 
     return parser.parse_args()
 
 
+def call_cx_freeze(gui_path, moz_path, python_lib_path):
```

Comment:
Ok, I will try it.
Ah, one more thing, would you mind offer the *args document to me again? I found it's very interesting and powerful and I would like to read it in detail  :)
Thank you very much!